src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -153,10 +154,33 @@ public Response describeManageDataAccessRequests(@Auth AuthUser authUser) {
     @GET
     @Produces("application/json")
     @Path("/manage/v2")
-    @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER})
-    public Response describeManageDataAccessRequestsV2(@Auth AuthUser authUser) {
-        List<DataAccessRequestManage> dars = dataAccessRequestService.describeDataAccessRequestManageV2(authUser);
-        return Response.ok().entity(dars).build();
+    @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER, SIGNINGOFFICIAL})
+    public Response describeManageDataAccessRequestsV2(@Auth AuthUser authUser, @QueryParam("roleName") Optional<String> roleName) {
+        try {
+            User user = userService.findUserByEmail(authUser.getName());
+            String roleNameValue = roleName.orElse(null);
+            if (Objects.nonNull(roleNameValue)) {
+                if (roleNameValue.equalsIgnoreCase(UserRoles.MEMBER.getRoleName())
+                || roleNameValue.equalsIgnoreCase(UserRoles.CHAIRPERSON.getRoleName())
+                || roleNameValue.equalsIgnoreCase(UserRoles.DATAOWNER.getRoleName())
+                || roleNameValue.equalsIgnoreCase(UserRoles.RESEARCHER.getRoleName())
+                || roleNameValue.equalsIgnoreCase(UserRoles.ALUMNI.getRoleName())) {
+                    throw new BadRequestException("Signing Official Role is the only role supported at this time");
+                }
```

Comment:
I generally try to stay away from long conditionals like this. What ends up happening is that over time, we insert variations into the conditions that make it more complicated than it needs to be. There are lots of ways to do this that are a little cleaner.

### Streaming
```
boolean valid = EnumSet.allOf(UserRoles.class)
        .stream()
        .map(UserRoles::getRoleName)
        .map(String::toLowerCase)
        .anyMatch(roleNameValue::equalsIgnoreCase);
```

## Collection contains
```
boolean valid = EnumSet.allOf(UserRoles.class)
        .stream()
        .map(UserRoles::getRoleName)
        .map(String::toLowerCase)
        .collect(Collectors.toList())
        .contains(roleNameValue.toLowerCase());
```

## Enum Values Array
```
boolean exists = Arrays
        .stream(UserRoles.values())
        .anyMatch(ur -> ur.getRoleName().equalsIgnoreCase(roleNameValue));
```